packages/shared/src/utils/helpers/duration.ts

```diff
@@ -120,10 +173,10 @@ export const hasIntradayDurationUnit = duration_units_list =>
  * @param {String} expiry_type
  * @returns {*}
  */
-export const resetEndTimeOnVolatilityIndices = (symbol, expiry_type) =>
+export const resetEndTimeOnVolatilityIndices = (symbol: string, expiry_type: string) =>
     /^R_/.test(symbol) && expiry_type === 'endtime' ? toMoment(null).format('DD MMM YYYY') : null;
 
-export const getDurationMinMaxValues = (duration_min_max, contract_expiry_type, duration_unit) => {
+export const getDurationMinMaxValues = (duration_min_max, contract_expiry_type: string, duration_unit: string) => {
```

Comment:
in case it's hard to figure out what is inside the `duration_min_max` object from the code itself, we can easily figure it out by launching the app, selecting a contract duration in Dtrader and console.log()ging its value 🙂